resend/actions/submitEmail.ts

```diff
@@ -0,0 +1,34 @@
+import { AppContext } from "../mod.ts";
+import { CreateEmailOptions, CreateEmailResponse } from "../utils/types.ts";
+
+export interface Props {
+  payload: CreateEmailOptions;
+}
+
+const action = async (
+  {
+    payload,
+  }: Props,
+  _req: Request,
+  ctx: AppContext,
+): Promise<CreateEmailResponse> => {
+  const { apiWrite, emailFrom, emailTo, subject } = ctx;
+  const emailFromDefault = `${emailFrom?.name ?? "Contact"} ${
+    emailFrom?.domain ?? "<onboarding@resend.dev>"
```

Comment:
those default fields can't be apps props?

"defaultDomain" "defaultFrom" ?

Reply:
And they are! Here is a fallback of the fallback 😁, if it is not sent by the action, and if the app's props are not filled in. Do you think it should be mandatory to fill in app props?